html/js/recorder.js

```diff
@@ -109,12 +109,11 @@
     },
 
     getBlob: function(name) {
-      var base64Data = Recorder.getBase64(name).split(',')[1];
-      return base64toBlob(base64Data, RECORDED_AUDIO_TYPE);
+      return Recorder.recorder.getWAV(name);
```

Comment:
I've just checked this and it seems that now, `getBlob(name)` returns simply plain javascript object without any data:
```
	{
		bytesAvailable: 0,
		endian: "littleEndian",
		length: 92204,
		objectEncoding: 3,
		position: 92204,
		shareable: false
	}
```